deploy/services-demo/README.md

```diff
@@ -0,0 +1,72 @@
+# How to run `wire-server` with "fake" AWS dependencies
+
+This document assumes that you have already compiled all services (i.e., you ran `make services` from the top level folder) and now you want to see how it all fits together.
```

Comment:
Perhaps you could add a line or two explaining that this is a "demo" setup (all data stored in-memory only)?

Reply:
Good idea, yes.